docs/source/WritingTests.rst

```diff
@@ -960,6 +960,10 @@ Avocado offers some options for the test writers to skip a test:
 Test ``skip()`` Method
 ----------------------
 
+.. warning:: `self.skip()` will be deprecated by the end of 2017.
```

Comment:
Nitpick: on English lessons we learned that `by ...` means sometimes before but close to the $DATE and `on ...` means exactly on the $DATE. Shouldn't this be `at the end of 2017`? (not that it'd matters, I'm just wondering whether my English is up to date and how it works in the real life...